recipes/vaex-core/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "vaex-core" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "07633b513081f27a2eb266a419914961c24f517885b8c1a3e572a3ddac7a196f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+     - vaex = vaex.__main__:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.11
```

Comment:
Do we need `numpy` at build time?

Reply:
Yes, for the c extension, needs the headers.

Reply:
Ah ok. Could you please add `toolchain` to `build` as well then?

Reply:
Out of interest (and because i'm trying to autogenerate recipes), when and why does toolchain need to be added?

Reply:
If there is something being compiled, the `toolchain` should be added. It ensures various flags get passed along like where headers and libraries should be searched for, how to set the RPATH, etc..

Reply:
Also as this is linking to `numpy`, would change this to something like `numpy 1.11.*` to ensure that the minimum workable version of `numpy` is used in the build. Otherwise if it gets built with a more recent version of `numpy` (e.g. 1.13) and a user tries to use it with `numpy` 1.11, there may be missing C API features that will cause issues.

Does this require `numpy` 1.11 as a minimum or can it work with older versions?

Reply:
It does require 1.11, but I might require 1.13 soon for the ufunc overriding (https://docs.scipy.org/doc/numpy-1.13.0/neps/ufunc-overrides.html). I thought the conda-forge packages were built with a specific numpy version, e.g. looking at for instance https://anaconda.org/conda-forge/h5py/files I np111-np113 etc, is this changing?
And doing 1.11.* would mean it would not build with 1.13.* right?

Reply:
I think the latest commit is what you ment, replaced build by `numpy 1.11.*` while run is still `numpy >= 1.13`.